src/airstack/generic.py

```diff
@@ -225,3 +234,465 @@ def remove_unused_variables(document_ast, query):
             if query.count(_variable.variable.name.value) == 1:
                 del document_ast.definitions[0].variable_definitions[_count]
     return print_ast(document_ast)
+
+
+def format_poaps_data(poaps, existing_user=None):
```

Comment:
as a good practise, it is good to give methods signature and description section.

Example:
```
def add_numbers(num_one: int, num_two:int) -> int:
"""
adds two numbers num_one and num_two and returns their sum
"""    
    return num_one+num_two
```